tests/utils.py

```diff
@@ -135,13 +134,16 @@ def no_spark_jobs(service_name):
     return len(driver_ips) == 0
 
 
-def _get_spark_options(options = None):
+def _get_spark_options(options, use_hdfs):
     if options is None:
         options = {}
 
-    if hdfs_enabled():
+    if use_hdfs:
         options["hdfs"] = options.get("hdfs", {})
         options["hdfs"]["config-url"] = "http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints"
+        options["security"] = options.get("security", {})
```

Comment:
So this defaults to always using `security`? What if the services are not kerberized?

Reply:
@NimaVaziri Current plan is to test only against kerberized HDFS, and not un-kerberized HDFS.

Reply:
Ah, alright then.